apps/web/screens/ApiCatalogue/ApiCatalogue.tsx

```diff
@@ -244,7 +244,7 @@ const ApiCatalogue: Screen<ApiCatalogueProps> = ({
               items={navigationItems}
               title={nn('linkThrounText')}
               titleLink={{
-                href: linkResolver('developerspage').as,
+                href: linkResolver('developerspage').href,
               }}
```

Comment:
The wrapping object is technically not needed here, we might also write this like: ```titleLink={ linkResolver('developerspage')}```

Reply:
this object is unrelated to next.js so i don't want to be spreading the linkresolver object here, just cherrypicking the href string